bucket/fnm.json

```diff
@@ -0,0 +1,22 @@
+{
+    "version": "1.22.8",
+    "description": "🚀 Fast and simple Node.js version manager, built in Rust",
```

Comment:
- Who ever think it is good idea to use emojis in application description??
- Source code language is completely irrelevant

Reply:
I've just copied the description straight from the original repo.
I've now editor the description